nx/source/runtime/argv.c

```diff
@@ -136,6 +140,12 @@ void argvSetup(void)
         __system_argc++;
     }
 
+    if (strncmp(__system_argv[__system_argc - 1], "NXLINKED", 8) == 0 ) {
```

Comment:
Maybe check for __system_argc>=1?

Reply:
Maybe use __system_argv[0]

Reply:
But that will break using cwd at switch/.